torchbenchmark/util/backends/jit.py

```diff
@@ -27,7 +34,10 @@ def _torchscript():
             module = torch.jit._script_pdt(module, example_inputs=[example_inputs, ])
         else:
             module = torch.jit.script(module, example_inputs=[example_inputs, ])
-        if model.test == "eval" and not backend_args.no_ofi:
+        if model.test == "eval" and backend_args.fuser != "fuser3" and not backend_args.no_ofi:
             module = torch.jit.optimize_for_inference(module)
+        if model.test == "eval" and backend_args.fuser == "fuser3":
```

Comment:
@davidberard98 can you help confirm this is the correct way of using `fuser3`?

Reply:
it seems reasonable, but maybe it wold be better to add a separate option for `jit.freeze` since it is generally applicable for any fuser? Also, jit.freeze doesn't necessarily support all torchscript features so it's possible that it would fail for some models. Just curious @chuanqi129, is freezing required for onednn?

Reply:
Thanks @davidberard98, you're right, `jit.freeze` is generally applicable for any fuser with `eval` test, however, the `jit.ofi` also include `jit.freeze`, so I think we may don't need add a separate option for it. How do you think? 

No, oneDNN fuser isn't freezing required. I did some code change to merge the `jit.freeze` with `jit.ofi` into parameter `--no-ofi`, do you think it is a good solution?

```
$ python run.py alexnet -d cpu -m jit   --fuser fuser0  -t eval
Running eval method from alexnet on cpu in jit mode with input batch size 128.
CPU Total Wall Time:  70.924 milliseconds
Correctness:                         True
$ python run.py alexnet -d cpu -m jit   --fuser fuser0  -t eval --no-ofi
Running eval method from alexnet on cpu in jit mode with input batch size 128.
CPU Total Wall Time:  93.563 milliseconds
Correctness:                         True
$ python run.py alexnet -d cpu -m jit   --fuser fuser3  -t eval
Running eval method from alexnet on cpu in jit mode with input batch size 128.
CPU Total Wall Time:  64.205 milliseconds
Correctness:                         True
$ python run.py alexnet -d cpu -m jit   --fuser fuser3  -t eval --no-ofi
Running eval method from alexnet on cpu in jit mode with input batch size 128.
CPU Total Wall Time: 101.545 milliseconds
Correctness:                         True
```

Thanks @xuzhao9 & @davidberard98 for the quick pre-review comments, very appreciate~

Reply:
The changes look good to me, merged. Thanks!